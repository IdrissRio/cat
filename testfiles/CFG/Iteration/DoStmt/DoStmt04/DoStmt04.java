public class DoStmt04 {
  {
    boolean b = true;
    do {
    } while (b);
    do {
    } while (b);
  }
}