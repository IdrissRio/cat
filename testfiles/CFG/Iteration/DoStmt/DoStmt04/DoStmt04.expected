NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [BooleanLiteral[39]]
NodeName: BooleanLiteral[39] Successors: [VariableDeclarator[36]]
NodeName: VariableDeclarator[36] Successors: [VarAccess[43]]
NodeName: VarAccess[43] Successors: [VarAccess[43], VarAccess[47]]
NodeName: VarAccess[47] Successors: [VarAccess[47], Exit[8]]
NodeName: Exit[8] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [VarAccess[47]]
NodeName: VarAccess[47] Predecessor: [VarAccess[43], VarAccess[47]]
NodeName: VarAccess[43] Predecessor: [VariableDeclarator[36], VarAccess[43]]
NodeName: VariableDeclarator[36] Predecessor: [BooleanLiteral[39]]
NodeName: BooleanLiteral[39] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
